packages/core/src/families/casper/serializer.ts

```diff
@@ -0,0 +1,30 @@
+import BigNumber from "bignumber.js";
+import type { RawCasperTransaction, CasperTransaction } from "./types";
+
+export const serializeCasperTransaction = ({
+  amount,
+  recipient,
+  family,
+  fees,
+  transferId,
+}: CasperTransaction): RawCasperTransaction => ({
+  amount: amount.toFixed(),
```

Comment:
We use `toString` on all other serialize functions, not sure why we would change it here
```suggestion
  amount: amount.toString(),
```

Reply:
Looking a LL code I see they do this in the families